phpBB/styles/prosilver/theme/bidi.css

```diff
@@ -502,10 +502,15 @@ li.breadcrumbs span:first-child > a {
 
 .rtl blockquote cite {
 	/* Username/source of quoter */
-	margin-right: 20px;
 	margin-left: 0;
 }
 
+.rtl blockquote cite:before, .uncited:before {
+	padding-right: 0;
+	padding-left: 5px;
+	content: '\f10e'; /* Font Awesome quote-right */
```

Comment:
no move it its the same concept of colors for rtl they are still found in colors.css and not bidi.css(why TF is this not just called rtl.css)

@vinny any issues with this from styles perspective?